app/src/main/java/org/coepi/android/domain/symptomflow/SymptomStep.kt

```diff
@@ -4,5 +4,5 @@ enum class SymptomStep {
     COUGH_TYPE, COUGH_DAYS, COUGH_DESCRIPTION,
     BREATHLESSNESS_DESCRIPTION,
     FEVER_DAYS, FEVER_TEMPERATURE_TAKEN_TODAY, FEVER_TEMPERATURE_SPOT,
-    FEVER_TEMPERATURE_SPOT_INPUT, FEVER_HIGHEST_TEMPERATURE
+    FEVER_TEMPERATURE_SPOT_INPUT, FEVER_HIGHEST_TEMPERATURE, EARLIEST_SYMPTOM_DATE
```

Comment:
Either remove "date" from this identifier or add it to the input class name / fragment / etc.